meta/meta.go

```diff
@@ -77,7 +74,6 @@ var (
 	mPolicyGlobalID   = []byte("PolicyGlobalID")
 	mPolicyMagicByte  = CurrentMagicByteVer
 	mDDLTableVersion  = []byte("DDLTableVersion")
-	mConcurrentDDL    = []byte("concurrentDDL")
```

Comment:
Do we need to remove the corresponding value stored in the TiKV cluster when upgrading? Although the space used is not much, it is better not to clean it up.

Reply:
We can just leave it here. Less code, less bug :)